### PR TITLE
 Update heketi vendor dependency to af26821e74991e7b7236f503e6310ec6b1d91f73

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@rootfs @childsb PTAL :)

File: glide.lock [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 - name: cloud.google.com/go
   version: 3b1ae45394a234c385be014e9a488f2bb6eef821
   subpackages:
   - compute/metadata
   - internal
+- name: github.com/asaskevich/govalidator
+  version: 7d2e70ef918f16bd6455529af38304d6d025c952
 - name: github.com/aws/aws-sdk-go
-  version: aace5875a5c3b85a3902c6d72b9caed301d64cce
+  version: 9b0098a71f6d4d473a26ec8ad3c2feaac6eb1da6
```
Comment (rootfs) [inline glide.lock]: why update aws sdk? 
run glide-vc and clean up unused update.

**Comment by humblec** [comment]
@rootfs @childsb @wongma7  Please review :).. Thanks

**Comment by humblec** [comment]
@rootfs PTAL :).. Thanks

**Comment by humblec** [comment]
Thanks @rootfs !

**Pull request merged**